### PR TITLE
Corrected Tile Hiding Criteria

### PR DESCRIPTION
**Pull request by clintonb**
This should allow changes of 0 to be displayed. Also, removed the validation setting from local.

@dsjen @rocha

### CODE REVIEW

File: analytics_dashboard/courses/templates/courses/enrollment_activity.html [hunk 1]
```diff
@@ -67,7 +67,7 @@ <h4 class="section-title">{% trans "Enrollment Metrics" %}</h4>
           </div>
 
           {% switch display_verified_enrollment %}
-            {% if summary.verified_enrollment %}
+            {% if not summary.verified_enrollment == None %}
```
Comment (rocha) [inline analytics_dashboard/courses/templates/courses/enrollment_activity.html:70]: How does operator precedence work in templates? Is this `(not variable) == None` or `not (variable == None)`?

Comment (clintonb) [inline analytics_dashboard/courses/templates/courses/enrollment_activity.html:70]: The latter. Django template syntax does not allow for `is not None`.

Comment (rocha) [inline analytics_dashboard/courses/templates/courses/enrollment_activity.html:70]: There is some discussion [here](http://stackoverflow.com/questions/3259279/django-templates), but I don't like any of the answers that much.

Comment (rocha) [inline analytics_dashboard/courses/templates/courses/enrollment_activity.html:70]: Seems fine then, I didn't know that `==` had higher precedence than `not` in python. It is usually backwards in other languages (C and Java at least). In python I am used to `is not None` like you said.

**Comment by rocha** [comment]
:rocket:

**Pull request merged**